implementations/utils/gpu.py

```diff
@@ -0,0 +1,457 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities to create, delete, and SSH with GPUs."""
+
+from __future__ import annotations
+
+
+from absl import logging
+import airflow
+from airflow.decorators import task
+from apis import gcp_config, test_config
+import fabric
+from google.api_core.extended_operation import ExtendedOperation
+from google.cloud import compute_v1
+from implementations.utils import ssh
+import io
+import paramiko
+import re
+import sys
+from typing import Iterable, Any
+import uuid
+
+
+def print_images_list(project: str) -> str:
```

Comment:
Could you help me understand what's the purpose of printing the image list? Is this for debugging purpose? Thanks!

Reply:
Remove the function since we use the predefined image.